ecosystem/sep-0024.md

```diff
@@ -758,13 +791,25 @@ Example response:
       "status": "completed",
       "amount_in": "500",
       "amount_out": "495",
```

Comment:
This should be 485. 
`amount_out = amount_in - amount_fee - refunds.amount_refunded - refunds.amount_fee`
485 = 500 - 5 - 10 - 0